packages/commonwealth/client/scripts/views/components/component_kit/CWBannerText/CWBannerText.scss

```diff
@@ -0,0 +1,17 @@
+.linkTextRow {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.link {
+  color: #3498db;
```

Comment:
use color from the pallete in colors.module.scss like here

![image](https://github.com/user-attachments/assets/6c4f161a-80bc-4739-8d06-4338769ba6f7)

Reply:
i used the  $primary-500;